lib/susedistribution.pm

```diff
@@ -160,7 +160,7 @@ sub x11_start_program {
     # too fast typing or looses characters because of the load caused (also
     # see below). See https://progress.opensuse.org/issues/18200
     if (check_var('DESKTOP', 'kde')) {
-        type_string_slow $program;
+        type_string_very_slow $program;
```

Comment:
Please don't. This is "very slow" and this is true. Slowing down even further *will* have a significant impact on all X11 KDE tests and should not done lightly. The ticket you are working on is for the wayland test on the virtio backend which has its own problems. Just typing very slow is not a good idea and will most likely not fix all issues.